presto-main/src/main/java/com/facebook/presto/execution/scheduler/nodeSelection/NodeSelectionStats.java

```diff
@@ -29,6 +29,8 @@
     private final CounterStat bucketedPreferredNodeSelectedCount = new CounterStat();
     private final CounterStat bucketedNonPreferredNodeSelectedCount = new CounterStat();
 
+    private final CounterStat deadNodeSkippedCount = new CounterStat();
```

Comment:
Not strong preference: should we rename this to preferredNodeNotAlive?

Reply:
Changed it to `preferredNonAliveNodeSkippedCount`